### DISCUSSION TITLE
Release 1.68.0 ?

### DISCUSSION THREAD

**Discussion by klockla** [grpc/grpc-java#11549] [Q&A] [answered]
Hello,

I'm wondering why there is a 1.68.0 version available in the public maven repo while it doesn't seem to be a release here and the related milestone is only complete at 50% ?

**Comment by ejona86** [comment, marked as answer]
The 1.67.0 release had the wrong version number inside, so it went to Maven Central as 1.68.0. We've adjusted the branches, but we're continuing with 1.67.x for now. The version numbers are aligned cross-language, so things will be awkward for ~4 more weeks, then we'll release 1.68.x for real as 1.68.1.
